config/locales/en.yml

```
@@ -702,8 +702,8 @@ doc_auth.instructions.text1: Other forms of ID are not accepted. We’ll check t
 doc_auth.instructions.text2: You will not need your physical SSN card.
 doc_auth.instructions.text3: We match your phone number with your personal information and send a one-time code to your phone.
 doc_auth.instructions.text4: Your password saves and encrypts your personal information.
-doc_auth.rate_limit_warning.plural_html: For security reasons, you have <strong>%{remaining_attempts} attempts</strong> remaining.
-doc_auth.rate_limit_warning.singular_html: For security reasons, you have <strong>1 attempt</strong> remaining.
+doc_auth.rate_limit_warning.plural_html: You can try <strong>%{remaining_attempts} more times.</strong> Then you must wait 6 hours before trying again.
+doc_auth.rate_limit_warning.singular_html: You can try <strong>1 more time.</strong> Then you must wait 6 hours before trying again.
```

Not introduced here, but we should probably be using [Rails default pluralization support here](https://guides.rubyonrails.org/i18n.html#pluralization)

```suggestion
doc_auth.rate_limit_warning_html.one: You can try <strong>1 more time.</strong> Then you must wait 6 hours before trying again.
doc_auth.rate_limit_warning_html.other: You can try <strong>%{count} more times.</strong> Then you must wait 6 hours before trying again.
```

Then we'd be able to simplify the logic:

https://github.com/18F/identity-idp/blob/0cb8aa37d25cde7d0219d52aaebff425c5c58336/app/presenters/socure_error_presenter.rb#L31-L35

..to:

```rb
t('doc_auth.rate_limit_warning_html', count: remaining_attempts)
```